docs/machine-learning/tutorials/object-detection-onnx.md

```diff
@@ -103,17 +102,17 @@ Now that you have a general understanding of what ONNX is and how Tiny YOLOv2 wo
 
 1. Download [The project assets directory zip file](https://github.com/dotnet/machinelearning-samples/raw/main/samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/assets.zip) and unzip.
 
-1. Copy the `assets` directory into your *ObjectDetection* project directory. This directory and its subdirectories contain the image files (except for the Tiny YOLOv2 model, which you'll download and add in the next step) needed for this tutorial.
+1. Copy the `assets` directory into your _ObjectDetection_ project directory. This directory and its subdirectories contain the image files (except for the Tiny YOLOv2 model, which you'll download and add in the next step) needed for this tutorial.
```

Comment:
I tend to agree with Tom that this set of changes seems odd. I looked through the parts of the file above, and I can't find existing examples of `_` for italics instead of `*`. I wonder why that style was preferred.

Reply:
@BillWagner The default setting for MD049 is *consistent*, and underscores are used on line 42 (at least).